webpack/assets/javascripts/react_app/common/urlHelpers.test.js

```diff
@@ -44,23 +35,27 @@ describe('urlWithSearch', () => {
 });
 
 describe('URI query and stringify tests', () => {
-  const visit = jest.fn();
   const baseHref = 'http://some-url.com/';
   const oldQuery = 'search=some-search&page=1&per_page=25';
   const href = `${baseHref}?${oldQuery}`;
 
+  beforeEach(() => {
+    mockWindowLocation({ href });
```

Comment:
Does it still need it here? Looks like the `visit` is the mock now

Reply:
Well the urlHelpers work with the `window.location.href` directly, so it is necessary here.